django/db/models/sql/where.py

```diff
@@ -108,7 +108,7 @@ def as_sql(self, compiler, connection):
                 # around the inner SQL in the negated case, even if the
                 # inner SQL contains just a single expression.
                 sql_string = 'NOT (%s)' % sql_string
-            elif len(result) > 1:
+            else:
```

Comment:
all the test failures seem related to this

Reply:
Without it there are failures on Postgres.

Reply:
the failures now are just assertions on generated SQL so pretty simple fixes